### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by karim-en**

### CODE REVIEW

**Comment by karim-en** [comment]
Releated #616

File: cli/init/eth-contracts.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const BN = require('bn.js')
 const fs = require('fs')
-const { Web3, normalizeEthKey } = require('rainbow-bridge-utils')
+const { Web3, normalizeEthKey, sleep } = require('rainbow-bridge-utils')
+
+const RETRY_SEND_TX = 15
```
Comment (sept-en) [inline cli/init/eth-contracts.js:5]: Where does this constant come from?

File: testing/transfer-eth-erc20/from-near.js [hunk 2]
```diff
@@ -299,19 +302,31 @@ class TransferEthERC20FromNear {
       const borshProofRes = borshifyOutcomeProof(proofRes)
       clientBlockHeight = new BN(clientBlockHeight)
       // Debugging output, uncomment for debugging.
-      // console.log(`proof: ${JSON.stringify(proofRes)}`);
-      // console.log(`client height: ${clientBlockHeight.toString()}`);
-      // console.log(`root: ${clientBlockMerkleRoot}`);
+      // console.log(`proof: ${JSON.stringify(proofRes)}`)
+      // console.log(`client height: ${clientBlockHeight}`)
```
Comment (sept-en) [inline testing/transfer-eth-erc20/from-near.js:306]: Remove commented-out code

File: testing/transfer-eth-erc20/from-near.js [hunk 3]
```diff
@@ -299,19 +302,31 @@ class TransferEthERC20FromNear {
...
+      // TODO investigate and fix this web3 call
+      // const ethERC20Contract = new web3.eth.Contract(
+      //   JSON.parse(fs.readFileSync(ethErc20AbiPath)),
+      //   ethErc20Address,
+      //   {
+      //     from: ethMasterAccount,
+      //     handleRevert: true
+      //   }
+      // )
+      // const oldBalance = await ethERC20Contract.methods.balanceOf(ethReceiverAddress).call()
```
Comment (sept-en) [inline testing/transfer-eth-erc20/from-near.js:321]: Remove commented-out code

File: testing/transfer-eth-erc20/from-near.js [hunk 4]
```diff
@@ -323,9 +338,9 @@ class TransferEthERC20FromNear {
           gasPrice: new BN(await robustWeb3.web3.eth.getGasPrice()).mul(new BN(ethGasMultiplier))
         }
       )
-      const newBalance = await ethERC20Contract.methods
-        .balanceOf(ethReceiverAddress)
-        .call()
+      // TODO investigate and fix this web3 call
+      // const newBalance = await ethERC20Contract.methods.balanceOf(ethReceiverAddress).call()
```
Comment (sept-en) [inline testing/transfer-eth-erc20/from-near.js:342]: What's about this TODO?

**Review by sept-en — APPROVED** [review]
Added some minor comments/questions, but others than that - LGTM! Thanks for fixing this!

File: cli/init/eth-contracts.js (hunk 1 above)
Comment (karim-en) [reply cli/init/eth-contracts.js:5]: It is just a magic number like [this](https://github.com/aurora-is-near/rainbow-bridge/blob/master/utils/robust.js#L8) and [this](https://github.com/aurora-is-near/rainbow-bridge/blob/fd7019ff001968105e8bb22627f5a5097ba345a8/utils/helpers.js#L5) to get around of nonce error.

File: testing/transfer-eth-erc20/from-near.js (hunk 4 above)
Comment (karim-en) [reply testing/transfer-eth-erc20/from-near.js:342]: There is an unknown reason why this call fail when loading the contract from the abi, I think this research is not urgent and could be fixed in a follow-up issue. So for now at least we should have a passed ci to detect buggy changes.

Comment (sept-en) [reply testing/transfer-eth-erc20/from-near.js:342]: Anyway, I think the `ethers` interface that you've used is better than loading the whole abi. Partial ABI is better, so please just remove this TODO then

**Pull request merged**